docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-enable-cluster-label-verification/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+description: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+title: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+menuTitle: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
+weight: 110
+---
+
+# Migrating Mimir Helm Installation to Use Memberlist Cluster Label
+
+## Introduction
```

Comment:
In general, we don't want stacked headings (headings with no content between them).

The introductory content can be moved out of the H2 and become the conceptual material at the start of this page.